torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -143,8 +143,9 @@ def resize_image_tensor(
         )
 
         if need_cast:
-            if interpolation == InterpolationMode.BICUBIC and dtype == torch.uint8:
-                image = image.clamp_(min=0, max=255)
+            # bicubic interpolation can overshoot
+            if interpolation == InterpolationMode.BICUBIC:
+                image = image.clamp_(min=0, max=_FT._max_value(dtype))
```

Comment:
I will need to check this internally before we merge. Shall we revert the commit and bring it separately form this PR to merge this faster?